api/index.js

```diff
@@ -21,3 +21,9 @@ module.exports = (request, response) => {
 
   middleware(request, response);
 };
+
+module.exports.config = {
+  api: {
+    bodyParser: false,
+  },
+};
```

Comment:
that didn't work, I still get the singnature problems

Reply:
https://vercel.com/guides/how-do-i-get-the-raw-body-of-a-serverless-function

It seems that you need another function

We have similar code in Octokit,https://github.com/octokit/webhooks.js/blob/d51daf8d3f2b50cd250e367fec85d62825268f13/src/middleware/node/get-payload.ts#L30-L45
I'm not sure why it isn't working though

Reply:
Thanks for the pointer! I'll investigate this another time: https://github.com/wip/app/issues/659